build.savant

```diff
@@ -40,11 +40,11 @@ project(group: "org.savantbuild", name: "savant-version", version: "2.0.0-RC.7",
 }
 
 // Plugins
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.6")
-java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
-javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.6")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
+def java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
+def javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
+def idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.7")
+def release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
```

Comment:
Not sure I understand the need for this. While this is technically "groovy" it is really supposed to be a Savant build file which doesn't necessarily have to follow general Groovy coding guidelines.

Reply:
Yes, fair point, discussed it and reverted.